### ISSUE TITLE
Vim new users installation TLDR

### ISSUE THREAD

**Issue by juanMarinero** [fannheyward/coc-texlab#209]
**TLDR** installation aimed for new Vim users.
Tested on _Ubuntu 20.04_ && [Vim 8.2](https://github.com/ycm-core/YouCompleteMe/wiki/Building-Vim-from-source) &&  [TeX Live 2020](https://www.tug.org/texlive/quickinstall.html)

Install [coc.nvim](https://github.com/neoclide/coc.nvim) plugin, _TexLab_, etc.
Open Vim and run `:CocInstall coc-texlab`.
On success, it shows the `package.json` path of `coc-texlab`. Open it searching for `textlab.path` (because [coc-texlab config](https://github.com/fannheyward/coc-texlab#configuration) already advise us to fulfill that empty field):
`$ vim /home/$USER/.config/coc/extensions/node_modules/coc-texlab/package.json -c "/texlab.path"`
And change its "default" with output of `$ whereis texlab`.

Partial **example** of final `package.json`:
```
"contributes": {
    "configuration": {
      "type": "object",
      "title": "TexLab configuration",
      "properties": {
        "texlab.path": {
          "type": "string",
          "default": "/home/WhateverUsername/.cargo/bin/texlab",
          "description": "Custom texlab binary path"
        },
```
